### PR TITLE
order scans newest first

### LINKED ISSUE #9: Scans list should have most recent scans on top

### PR DESCRIPTION
**Pull request by hannes-dev**
fixes #9

### CODE REVIEW

File: vingo/database/scans.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ var (
 )
 
 func GetScansForUser(user_id int) ([]Scan, error) {
-	scans_rows, err := db.Query("SELECT scan_time, card_serial FROM scans WHERE card_serial IN (SELECT serial FROM cards WHERE user_id = $1);", user_id)
```
Comment (Topvennie) [inline vingo/database/scans.go:34]: SQL uses ORDER BY

**Review by Topvennie — CHANGES_REQUESTED** [review]

**Review by FKD13 — CHANGES_REQUESTED** [review]
DESC

File: vingo/database/scans.go (hunk 1 above)
Comment (Topvennie) [inline vingo/database/scans.go:34]: Should be DESC

**Review by Topvennie — CHANGES_REQUESTED** [review]

File: vingo/database/scans.go (hunk 1 above)
Comment (Topvennie) [inline vingo/database/scans.go:34]: still the same problem
https://www.w3schools.com/sql/sql_intro.asp

**Review by Topvennie — CHANGES_REQUESTED** [review]

**Review by FKD13 — CHANGES_REQUESTED** [review]
Force push fixes next time pls

**Review by FKD13 — APPROVED** [review]
Missing tests but looks to me

**Review by Topvennie — APPROVED** [review]
Perfect, good job 👍

**Comment by FKD13** [comment]
good

**Comment by Reavershark** [comment]
0/0 tests passed 👍

**Review by Reavershark — APPROVED** [review]

**Pull request merged**